projects/dotcms-field-elements/src/components/dot-select/dot-select.e2e.ts

```diff
@@ -1,93 +1,276 @@
-                    <label for=\"dot-testName\">
-                        testLabel
-                    </label>
-                    <span class=\"dot-field__required-mark\">
-                        *
-                    </span>
-                    </div>
-                    <select id=\"dot-testName\">
-                        <option value=\"\"></option>
-                        <option value=\"1\">
-                            valueA
-                        </option>
-                        <option value=\"2\">
-                            valueB
-                        </option>
-                    </select>
-                    <span class=\"dot-field__hint\">
-                        testHint
-                    </span>
-            </dot-select>`);
+        describe('disabled', () => {
+            it('should not render attribute', async () => {
+                const htmlElement = await getSelect(page);
+                expect(await htmlElement.getAttribute('disabled')).toBe(null);
+            });
+
+            it('should render attribute', async () => {
+                element.setProperty('disabled', true);
+                await page.waitForChanges();
+                const htmlElement = await getSelect(page);
+                expect(await htmlElement.getAttribute('disabled')).toBe('');
+            });
+
+            it('should not break with invalid data', async () => {
+                element.setProperty('disabled', ['a', 'b']);
+                await page.waitForChanges();
+                const htmlElement = await getSelect(page);
+                expect(await htmlElement.getAttribute('disabled')).toBe('');
+            });
         });
 
-        it('should be invalid, touched & dirty and the error msg should display', async () => {
-            await page.select('select', '');
-            await page.waitForChanges();
-            expect(element).toHaveClasses(['dot-invalid', 'dot-touched', 'dot-dirty']);
+        describe('name', () => {
+            const value = 'test';
+
+            it('should render attribute in label and select', async () => {
+                element.setProperty('name', value);
+                await page.waitForChanges();
+
+                const selectElement = await getSelect(page);
+                const idValue = await selectElement.getProperty('id');
+                expect(idValue.indexOf(value)).toBeGreaterThan(-1);
+
+                const labelElement = await getDotLabel(page);
+                expect(await labelElement.getProperty('name')).toBe(value);
+            });
+
+            it('should not render attribute in label and select', async () => {
+                const selectElement = await getSelect(page);
+                const idValue = await selectElement.getProperty('id');
+                expect(idValue.indexOf(value)).toBe(-1);
+
+                const labelElement = await getDotLabel(page);
+                expect(await labelElement.getProperty('name')).toBe('');
```

Comment:
Double check if we have to use `getProperty` or `getAttribute`, because is different.